llvm/test/CodeGen/Hexagon/arg-copy-elison.ll

```diff
@@ -0,0 +1,39 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc -mtriple hexagon-- -O3 -o - %s | FileCheck %s
```

Comment:
Do you really need the -O3? It doesn't make a difference on most targets and -O2 is the default

Reply:
Looks like -O2 also reproduce the failure (while -O1 won't trigger the problem). I'll change it to -O2.

Reply:
Just drop the flag